src/db/migrations/20230703010356_add-profile-table.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export function up(knex) {
+  return knex.schema
+    .createTable('profiles', table => {
+      table.increments('id').primary()
+      table.integer('onerep_profile_id').references('subscribers.onerep_profile_id').nullable()
+      table.integer('first_name')
+      table.integer('last_name')
+      table.integer('city_name')
+      table.integer('state_code')
```

Comment:
Should these be `.integer()` or `.string()`?

```sh
git grep -Pho "table\.\w+\(" | sort | uniq

table.boolean(
table.date(
table.dropColumn(
table.dropColumns(
table.dropIndex(
table.dropTimestamps(
table.increments(
table.index(
table.integer(
table.jsonb(
table.renameColumn(
table.specificType(
table.string(
table.text(
table.timestamp(
table.timestamps(
```

Reply:
Yeah I noticed that yesterday when rebasing and re-testing from scratch, thanks